readthedocs/oauth/services/bitbucket.py

```diff
@@ -214,5 +214,5 @@ def setup_webhook(self, project):
             log.error('Bitbucket webhook creation failed for project: %s',
                       project)
             log.debug('Bitbucket webhook creation failure response: %s',
-                      dict(resp))
+                      resp.content)
```

Comment:
That only works for JSON content, and most of the issues here are the responses aren't valid JSON.